robot/Cumulus/tests/browser/gift_entry/review_donations_update_pmt_bge.robot

```diff
@@ -0,0 +1,93 @@
+    Set suite variable   &{PAYMENT}
+    ${ns} =              Get NPSP Namespace Prefix
+    Set suite variable   ${NS}
+
+*** Test Cases ***
+Review Donation And Update Payment For Batch Gift
+    [Documentation]                      Create an organization account with open opportunity (with payment record) via API. Go to SGE form
+    ...                                  select the donor as account and the account created. Verify review donations modal and select to update payment.
+    ...                                  Change date to today and payment amount to be less than opp amount. Verify that same payment record got updated
+    ...                                  with new amount and date but opportunity is still prospecting and amount is not updated.
+    [tags]                               unstable      feature:GE                    W-042803   
+    #verify Review Donations link is available and update a payment                           
+    Go To Page                           Landing                       GE_Gift_Entry         
+    Click Gift Entry Button              New Batch
+    Wait Until Modal Is Open
+    Select Template                      Default Gift Entry Template
+    Load Page Object                     Form                          Gift Entry
+    Fill Gift Entry Form
+    ...                                  Batch Name=${ACCOUNT}[Name]Automation Batch
+    ...                                  Batch Description=This is a test batch created via automation script
+    Click Gift Entry Button              Next
+    Click Gift Entry Button              Save
+    Current Page Should Be               Form                          Gift Entry
+    ${batch_id} =                        Save Current Record ID For Deletion     ${NS}DataImportBatch__c
+    Fill Gift Entry Form
+    ...                                  Donor Type=Account1
+    ...                                  Existing Donor Organization=${ACCOUNT}[Name]
+    Click Button                         Review Donations
+    Wait Until Modal Is Open
+    Click Button                         Update this Payment
+    Wait Until Modal Is Closed 
+    Fill Gift Entry Form                 
+    ...                                  Donation Amount=499.50
+    ...                                  Donation Date=Today
+    Click Button                         Save & Enter New Gift
+    #verify donation date and amount values changed on table
+    Verify Gift Count                    1
+    Verify Table Field Values            Batch Gifts
+    ...                                  Donor Name=${ACCOUNT}[Name]
+    ...                                  Donation Amount=$499.50
+    ...                                  Donation Name=${PAYMENT}[Name]
+    ...                                  Donation Date=${UI_DATE}
+    Scroll Page To Location              0      0
```

Comment:
is this needed ? can it be part of any of the keywords?

Reply:
I already have a scroll element into view in the next keyword but somehow in this particular test case it is not sure to top enough for the button to be visible. Only this option worked in this case. I cannot add this to the following keyword "Click Gift Entry Button" as this keyword is used for various buttons and the buttons could be anywhere on the page